wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html

```diff
@@ -50,25 +48,45 @@ <h2>{% trans 'Pages you can moderate' %}</h2>
                                     </ul>
                                 {% endif %}
                             </td>
-                            <td class="task" valign="top">
-                                {{ state.task.name }}
-                            </td>
-                            <td valign="top">
-                                <div class="human-readable-date" title="{{ state_type_started_at|date:"d M Y H:i" }}">{% blocktrans with time_period=state.task_type_started_at|timesince %}{{ time_period }} ago{% endblocktrans %} </div>
+                            <td class="tasks" valign="top">
+                                {% for task in workflow_tasks %}
+                                    <span data-wagtail-tooltip="{{ task.name }}: {{ task.status_display }}">
+                                        {% if task.status == 'approved' %}
+                                            {% icon "success" title=task.status_display class_name="uniform" %}
+                                        {% elif task.status == 'rejected' %}
+                                            {% icon "error" title=task.status_display class_name="uniform" %}
+                                        {% else %}
+                                            {% icon "radio-empty" title=status_display class_name="uniform" %}
+                                        {% endif %}
+                                    </span>
+                                {% endfor %}
                             </td>
                             <td valign="top">
-                                <div class="human-readable-date" title="{{ revision.created_at|date:"d M Y H:i" }}">{% blocktrans with time_period=revision.created_at|timesince %}{{ time_period }} ago{% endblocktrans %} </div>
+                                <div class="human-readable-date" title="{{ state.task_type_started_at|date:"d M Y H:i" }}">{% blocktrans with time_period=state.task_type_started_at|timesince %}{{ time_period }} ago{% endblocktrans %} </div>
```

Comment:
I think at this stage `task_type_started_at` can be removed on the model, and we can just use the field instead - it was necessary when we copied task states between revisions, but that's no longer needed.

Reply:
Which field are you referring to?

Reply:
Yes, that's the one

Reply:
This also makes it work better with resubmission (currently if I submit on Monday, reject on Tuesday and resubmit on Wednesday for example, my page will say it's been waiting to be reviewed since Monday, rather than Wednesday)